### PR TITLE
Roster

### PR DESCRIPTION
**Pull request by sofiaroz**
- must reseed

### CODE REVIEW

File: client/src/components/Profile.js [hunk 1]
```diff
@@ -63,83 +68,124 @@ const Profile = () => {
...
+                </Col>
+              </Row>{' '}
+              <Row type="flex">
+              <Col>
+                <p>
+                  <span className="profileItem" id="semestersAttendedBubble">
+                    SEMESTERS
+                  </span>{' '}
+                  {/* {displayStudentSemesters} */}
+                </p>
```
Comment (emilyzwang) [inline client/src/components/Profile.js:127]: do we want to display semesters here?

File: client/src/components/RosterCard.js [hunk 2]
```diff
@@ -41,99 +78,261 @@ const RosterCard = props => {
...
+      description = (
+        <div>
+          <h2>{name}</h2>
+          <p>
+            <span className="rosterCardItem" id="emailBubble">
+              EMAIL
+            </span>{' '}
+            {email}
+          </p>
+          <p>
```
Comment (emilyzwang) [inline client/src/components/RosterCard.js:189]: + fav candy

File: client/src/components/RosterCard.js [hunk 3]
```diff
@@ -1,37 +1,74 @@
...
+
+    setEditedStudentSemesters(edited);
+  };
+
   const editStudentProfile = () => {
-    if (editedNotes.length >= 255) {
+    let updateNotes = editedNotes;
+    if (!updateNotes) {
+      updateNotes = notes;
+    }
```
Comment (emilyzwang) [inline client/src/components/RosterCard.js:55]: issues if user is trying to delete notes or if `notes` is null

File: controllers/rosters_controller.js [hunk 4]
```diff
@@ -10,29 +10,59 @@ const getUsersBySite = async (req, res) => {
...
       .where('site_id', siteid[0].site_id)
       .where('role', roleType);
     res.status(200).send(data);
   } catch (error) {
+    console.log(error);
     res.status(500).json({ error });
   }
 };
+
+const getUserSemester = async (req, res) => {
```
Comment (emilyzwang) [inline controllers/rosters_controller.js:35]: do we use this?

Comment (sofiaroz) [reply controllers/rosters_controller.js]: - got rid of this after collapsing semesters column

**Comment by sofiaroz** [comment]
- collapsed semesters column from user_site (previously, user_semester_site) table into studentSemesters in user table
- updated migration files
- renamed user_semester_site table appropriately to user_site (note: file names have not been updated, to update file names after merging all PRs to prevent conflicts)

**Pull request merged**